src/website/src/app/utils/code-snippet.ts

```diff
@@ -10,10 +10,10 @@ import { CodeHighlight } from './code-highlight';
   selector: 'clr-code-snippet',
   template: `
         <ng-container *ngIf="!disablePrism">
-            <pre><code [clr-code-highlight]="'language-'+language">{{code.default.trim()}}</code></pre>
+            <pre><code [clr-code-highlight]="'language-'+language">{{code.trim()}}</code></pre>
         </ng-container>
         <ng-container *ngIf="disablePrism">
-            <pre><code class="clr-code">{{code.default.trim()}}</code></pre>
+            <pre><code class="clr-code">{{code.trim()}}</code></pre>
```

Comment:
LGTM, qq: if we have string trimming every time, can we get that in the `code` getter?